src/js-library/screen-sharing.md

```diff
@@ -102,20 +101,57 @@ More info. on inline installation can be found at: [https://developer.chrome.com
 ##Adding Screen Sharing to Your Application
 Once you have SSL working and have your Chrome Extension in place, it's time for the fun stuff: the code!
 
-Adding screen sharing to your application is actually quite easy. The first step is creating a configuration object which specifies the capabilities that you will allow for your stream. Since Audio is not yet supported for screen sharing, we recommend the following settings:
+Adding screen sharing to your application is actually quite easy. You need a reference to the Endpoint you want to share your screen with
 
 ```
-var capabilities = {audio: false, video: true};
+var recipientEndpoint = client.getEndpoint({ id: recipientId });
 ```
 
 
 
+a `<video>` object to attach the stream to
 
+```
+<video id="videoElement"></video>
+```
+
+
+and a listener for the call event.
+
+```
+client.listen('call', function(evt) {
+    activeCall = evt.call;
 
+    // We only want to answer if we didn't initiate the call
+    if(activeCall.caller !== true) {
+        var videoElement   = document.getElementById('videoElement');
 
+        activeCall.answer({
+            videoLocalElement: videoElement,
+            videoRemoteElement: videoElement
```

Comment:
Probably don't want to use the same element here, if just to illustrate that they'll usually be different.